iot/api-client/end-to-end-example/pom.xml

```diff
@@ -85,6 +85,11 @@
       <artifactId>commons-cli</artifactId>
       <version>1.4</version>
     </dependency>
+    <dependency>
```

Comment:
Gson should be used instead, per security team advice.

see go/json-madness

Reply:
ok, thanks!
will do later.